docs/usage.rst

```diff
@@ -69,6 +69,82 @@ callable is called, or to prevent it, use the ``call_default`` parameter:
     Note that ``call_default`` is only used when the related setting is missing
     from the project settings!
 
+Inherit from environment variables
+''''''''''''''''''''''''''''''''''
+
+Nowadays it became more and more popular to inherit settings from the environment.
+This functionality is supported as well. If the setting name is found in ``os.environ``,
+this value is used. To distinguish between different setting types, the value is
+parsed as a JSON by default. Some specific setting types support even other, easier
+and nicer, formats. For more information see the example below.
```

Comment:
By default, all values are parsed as JSON, see other supported values in description of individual setting classes.

Reply:
Done.